ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -1,4 +1,57 @@
 periodics:
+- agent: kubernetes
+  cluster: build03
+  cron: 0 */6 * * 0-5
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "false"
+  name: osde2e-hypershift-stage
```

Comment:
is there any reason we aren't using prowgen for this job?

Reply:
Answered in our call, but I'll set something up to show you our current problem with handcrafted vs prowgen.